lgc/interface/lgc/Builder.h

```diff
@@ -196,6 +197,11 @@ class Builder : public BuilderCommon {
   // Record shader modes into IR metadata if this is a shader compile (no PipelineState).
   virtual void recordShaderModes(llvm::Module *module) {}
 
+  // Find the resource node for the given set,binding
+  virtual const ResourceNode *findResourceNode(ResourceNodeType nodeType, unsigned descSet, unsigned binding) const {
+    return nullptr;
+  }
```

Comment:
Since BuilderReplayer no longer derives from Builder, I think you can make this a pure virtual method now (` = 0`)

Reply:
The empty func here is trying to avoid to implement function in the builderImpl and its subsequent classes like inoutBuilder.
Make it pure virtual would break the compiling here.